packages/office-addin-cli/test/test.json

```diff
@@ -1,10 +1,10 @@
 {
     "scripts": {
-      "test": "1",
-      "test-test": "2",
-      "test_test": "3",
-      "test test": "4",
-      "test-tes-te": "5"
+      "one": "1",

```

Comment:
There was a problem with the test using "test" for the script name. It turns out that when "npm test" is run, it was picking up the value from office-addin-cli package.json which contains a script named "test". So I changed to use names which would not exist.